upgrade-operator/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "upgrade-operator"
+description = "Upgrade Operator"
+version = "0.1.0"
+edition = "2021"
+authors = ["Sahil Raja <sahil.raja@datacore.com>"]
+
+[[bin]]
+name = "upgrade-operator"
+path = "controller/src/bin/main.rs"
+
+[lib]
+name = "components"
```

Comment:
components seems to be a very open ended name to a library.. what about operator?

Reply:
Have used common instead of components.

Reply:
common also could be confusing because if other crates need to access the upgrade operator they'd use `common::SomeType` which doesn't make sense for them because it's only common for the upgrade operator itself right?
out of these 3, operator would be my choice.

Reply:
Okay. 
So name should be operator and directory name can be common?

Reply:
The rust standard is simply `src/lib.rs` but it's up to you.

Reply:
Okay. Will do changes of the layout while implementing different modules if required.